config/paging.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * Paging Config
+ */
+return [
```

Comment:
I think we could just keep them as constants. Not sure if there's a reason to keep them as environmental variables. They will be the same regardless of the environment, right?